src/Accounts/Accounts/Rest/InvokeAzRestMethodCommand.cs

```diff
@@ -95,14 +95,14 @@ public class InvokeAzRestMethodCommand : AzureRMCmdlet
         [Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
         public SwitchParameter AsJob { get; set; }
 
-        [Parameter(Mandatory = false, HelpMessage = "Wait till the long-running operation completes before returning the result.")]
-        public SwitchParameter WaitTillCompletion { get; set; }
+        [Parameter(Mandatory = false, HelpMessage = "Waits for the long-running operation completes before returning the result.")]
+        public SwitchParameter WaitForCompletion { get; set; }
 
-        [Parameter(Mandatory = false, HelpMessage = "Specifies where to poll for long-running operation status.")]
+        [Parameter(Mandatory = false, HelpMessage = "Specifies the polling header (to fetch from) for long-running operation status.")]
```

Comment:
Please update the parameter description of `PollFrom` and `FinalResultFrom` in the markdown file. They won't be refreshed by regenerating docs.